docs/CODEOWNERS

```diff
@@ -0,0 +1,145 @@
+# Archipelago World Code Owners / Maintainers Document
+#
+# This file is used to notate the current "owners" or "maintainers" of any currently merged world folder. For any pull
+# requests that modify these worlds, a code owner must approve the PR in addition to a core maintainer. This is not to
+# be used for files/folders outside the /worlds folder, those will always need sign off from a core maintainer.
+#
+# All usernames must be GitHub usernames (and are case sensitive).
+
+# Adventure
+/worlds/adventure/ @JusticePS
+
+# A Link to the Past
+/worlds/alttp/ @Berserker66
+
+# ArchipIDLE
+/worlds/archipidle/ @LegendaryLinux
+
+# Sudoku (BK Sudoku)
+/worlds/bk_sudoku/ @Jarno458
+
+# Blasphemous
+/worlds/blasphemous/ @TRPG0 @BrandenEK
+
+# Bumper Stickers
+/worlds/bumpstik/ @FelicitusNeko
+
+# ChecksFinder
+/worlds/checksfinder/ @jonloveslegos
+
+# Clique
+/worlds/clique/ @ThePhar
+
+# Dark Souls III
+/worlds/dark_souls_3/ @Marechal-L
+
+# Donkey Kong Country 3
+/worlds/dkc3/ @PoryGone
+
+# DLCQuest
+/worlds/dlcquest/ @axe-y @agilbert1412
+
+# Factorio
+/worlds/factorio/ @Berserker66
+
+# Final Fantasy
+/worlds/ff1/ @jtoyoda
+
+# Hollow Knight
+/worlds/hk/ @dewiniaid
+
+# Hylics 2
+/worlds/hylics2/ @TRPG0
+
+# Kingdom Hearts 2
+/worlds/kh2/ @JaredWeakStrike
+
+# Links Awakening DX
+/worlds/ladx/ @zig-for
+
+# Lufia II Ancient Cave
+/worlds/lufia2ac/ @el-u @wordfcuk
+
+# Meritous
+/worlds/meritous/ @FelicitusNeko
+
+# The Messenger
+/worlds/messenger/ @alwaysintreble
+
+# Minecraft
+/worlds/minecraft/ @KonoTyran @espeon65536
+
+# Noita
+/worlds/noita/ @ScipioWright @heinermann
+
+# Ocarina of Time
+/worlds/oot/ @espeon65536
+
+# Ori and the Blind Forest
+/worlds/oribf/ #Unmaintained
+
+# Overcooked! 2
+/worlds/overcooked2/ @toasterparty
+
+# Pokemon Red and Blue
+/worlds/pokemon_rb/ @Alchav
+
+# Raft
+/worlds/raft/ @SunnyBat
+
+# Rogue Legacy
+/worlds/rogue_legacy/ @ThePhar
+
+# Risk of Rain 2
+/worlds/ror2/ @kindasneaki
+
+# Sonic Adventure 2 Battle
+/worlds/sa2b/ @PoryGone @RaspberrySpace
+
+# Starcraft 2 Wings of Liberty
+/worlds/sc2wol/ @Ziktofel
+
+# Super Metroid
+/worlds/sm/ @lordlou
+
+# Super Mario 64
+/worlds/sm64ex/ @N00byKing
+
+# Super Mario World
+/worlds/smw/ @PoryGone
+
+# SMZ3
+/worlds/smz3/ @lordlou
+
+# Secret of Evermore
+/worlds/soe/ @black-sliver
+
+# Slay the Spire
+/worlds/spire/ @KonoTyran
+
+# Stardew Valley
+/worlds/stardew_valley/ @Jouramie @agilbert1412
```

Comment:
In a context where we give credit, I'd add Jouramie as a very significant contributor.
But in this context, where the goal is to have someone to contact about the world, I would set myself up as the only name for Stardew Valley (for now). Jouramie has not been actively contributing to the project since version 2.x.x and it is unclear if he plans on contributing again.

Reply:
Sure, if that changes let me know and I can update accordingly.